### PR TITLE
Translation of seo chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of #586

I've also updated src/content/en/2021/seo.md
Please confirm it too.

### CODE REVIEW

File: src/content/en/2021/seo.md [hunk 1]
```diff
@@ -571,21 +571,21 @@ Heading elements (`<h1>`, `<h2>`, etc.) are an important structural element. Whi
 {{ figure_markup(
    image="heading-element-usage.png",
    caption="Heading element usage.",
-   description="Bar chart showing the percent of pages with the presence of H elements by heading tag (level 1, 2, 3, 4). There was little to no difference between desktop and mobile results. `h1` headings were found on 65.4% of pages, `h2`s were found the most frequently on 71.9% of pages, `h3`3s were found on 61.8% of pages and `h4` headings were found on 37.6% of pages.",
+   description="Bar chart showing the percent of pages with the presence of H elements by heading tag (level 1, 2, 3, 4). There was little to no difference between desktop and mobile results. `h1` headings were found on 65.4% of pages, `h2` were found the most frequently on 71.9% of pages, `h3` were found on 61.8% of pages and `h4` headings were found on 37.6% of pages.",
```
Comment (tunetheweb) [inline src/content/en/2021/seo.md:574]: I think this reads better with the `s`. "h2s is basically short for h2 headings". The h3 one was incorrect though.

```suggestion
   description="Bar chart showing the percent of pages with the presence of H elements by heading tag (level 1, 2, 3, 4). There was little to no difference between desktop and mobile results. `h1` headings were found on 65.4% of pages, `h2`s were found the most frequently on 71.9% of pages, `h3`s were found on 61.8% of pages and `h4` headings were found on 37.6% of pages.",
```

File: src/content/en/2021/seo.md [hunk 2]
```diff
@@ -571,21 +571,21 @@ Heading elements (`<h1>`, `<h2>`, etc.) are an important structural element. Whi
...
    chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1197492338&format=interactive",
    sheets_gid="1347655296",
    sql_file="seo-stats.sql"
    )
 }}
 
-For main headings, more pages (71.9%) have `h2`s than have `h1`s (65.4%). There's no obvious explanation for the discrepancy. 61.4% of desktop and mobile pages use `h3`s and less than 39% use `h4`s.
+For main headings, more pages (71.9%) have `h2` than have `h1` (65.4%). There's no obvious explanation for the discrepancy. 61.4% of desktop and mobile pages use `h3` and less than 39% use `h4`.
```
Comment (tunetheweb) [inline src/content/en/2021/seo.md:581]:
```suggestion
For main headings, more pages (71.9%) have `h2`s than have `h1`s (65.4%). There's no obvious explanation for the discrepancy. 61.4% of desktop and mobile pages use `h3`s and less than 39% use `h4`s.
```

File: src/content/en/2021/seo.md [hunk 3]
```diff
@@ -571,21 +571,21 @@ Heading elements (`<h1>`, `<h2>`, etc.) are an important structural element. Whi
...
 
 There was very little difference between desktop and mobile heading usage, nor was there a major change versus 2020.
 
 {{ figure_markup(
    image="non-empty-heading-element-usage.png",
    caption="Non-empty heading element usage.",
-   description="Bar chart showing the percent of pages with the presence of non-empty `<h>` elements by heading tag (level 1, 2, 3, 4). There was little to no difference between desktop and mobile results. `h1` headings were found on 58.1% of pages, `h2`s were found the most frequently on 70.5% of pages, `h3`s on 60.3% of pages and `h4` headings were found on 36.5% of pages.",
+   description="Bar chart showing the percent of pages with the presence of non-empty `<h>` elements by heading tag (level 1, 2, 3, 4). There was little to no difference between desktop and mobile results. `h1` headings were found on 58.1% of pages, `h2` were found the most frequently on 70.5% of pages, `h3` on 60.3% of pages and `h4` headings were found on 36.5% of pages.",
```
Comment (tunetheweb) [inline src/content/en/2021/seo.md:588]:
```suggestion
   description="Bar chart showing the percent of pages with the presence of non-empty `<h>` elements by heading tag (level 1, 2, 3, 4). There was little to no difference between desktop and mobile results. `h1` headings were found on 58.1% of pages, `h2`s were found the most frequently on 70.5% of pages, `h3`s on 60.3% of pages and `h4` headings were found on 36.5% of pages.",
```

**Review by tunetheweb — APPROVED** [review]
Looks good. Thanks!

Going to revert some of the English changes as reads better with the `s`s. But the rest of the English changes are good.

File: src/content/ja/2021/seo.md [hunk 4]
```diff
@@ -0,0 +1,758 @@
...
+   description="パーセンタイル（10、25、50、75、90）ごとの未加工の可視ワード数を示す棒グラフ。デスクトップ用未加工ページの中央値は369語、モバイル用ページの中央値は321語でした。モバイル・ページで、すべてのパーセンタイルで未加工の単語数が少なくなっています。",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=58186900&format=interactive",
+   sheets_gid="455169599",
+   sql_file="seo-stats-by-percentile.sql"
+   )
+}}
+
+- デスクトップページの中央値は369語であるのに対し、2020年には360語になっています。
+- モバイルページの中央値は321語であるのに対し、2020年には312語となっています。
+- モバイルの未加工ページは、デスクトップのレンダリングページよりも13.1%少ない単語しか含まれていません。Googleはモバイル専用のインデックスであることに注意してください。モバイルHTML版にないコンテンツは、インデックスされない可能性があります。
```
Comment (tunetheweb) [inline src/content/ja/2021/seo.md:507]: Note in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2788/files we corrected an error in this line. It looks like this error is still here. Can you correct?

File: src/content/ja/2021/seo.md [hunk 5]
```diff
@@ -0,0 +1,758 @@
...
+- 中央のデスクトップとモバイルのページでは、`<img>` タグに `alt` 属性なしがゼロか、それと近い値になっています。これは、中央ページにある`<img>`タグの2～3%に`alt`属性がなかった2020年に比べて改善されています。
+
+#### 画像 `loading` 属性
+
+`<img>` 要素の `loading` 属性は、ユーザーエージェントがページ上の画像のレンダリングと表示をどのように優先させるかに影響します。ユーザー体験やページの読み込み性能に影響を与える可能性があり、これらはいずれもSEOの成功に影響を与えます。
+
+{{ figure_markup(
+   image="image-loading-property-usage.png",
+   caption="画像読み込みのプロパティの使用状況。",
+   description='ページと画像読み込みプロパティの使用率（missing, lazy, eager, invalid, auto, blank）を示す棒グラフ。当社のデータでは、デスクトップページの83.3%、モバイルページの83.5%で画像の読み込みプロパティが欠落していることが判明しました。その結果、デスクトップとモバイルのページの15.6%が `loading="lazy"` を使用しており、`loading="eager"` はデスクトップとモバイルのページの0.8%に過ぎないことがわかりました。その他の画像読み込みプロパティである `loading="invalid"` や `loading="auto"` 、 `loading="blank"` は、デスクトップやモバイルのページの1%未満にしか使用されていません。',
```
Comment (tunetheweb) [inline src/content/ja/2021/seo.md:453]: Can you correct this, following the recent correct in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2788/files ?

**Review by tunetheweb — CHANGES_REQUESTED** [review]
Actually spotted a couple of recent changes which have not been included in the translation. Could you have another look?

File: src/content/ja/2021/seo.md (hunk 4 above)
Comment (ksakae1216) [reply src/content/ja/2021/seo.md:507]: OK, I translated.

File: src/content/ja/2021/seo.md (hunk 5 above)
Comment (ksakae1216) [reply src/content/ja/2021/seo.md:453]: @tunetheweb Sorry, I don't understand.
What do I need to fix?

Comment (tunetheweb) [reply src/content/ja/2021/seo.md:453]: Oops. Wrong link: https://github.com/HTTPArchive/almanac.httparchive.org/pull/2787

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**